libraries/engage/locations/reducers/userSearch.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import * as Redux from 'redux';
+import appConfig from '@shopgate/pwa-common/helpers/config';
+import { STORE_SEARCH } from '../constants';
+
+export type UserSearchStateType = {
+  postalCode: string,
+  countryCode: string,
+};
+
+const defState: UserSearchStateType = {
+  postalCode: '',
+  countryCode: appConfig.marketId,
+};
+
+/**
+ * Stores the user's search input.
+ * @param {Object} state The current state.
+ * @param {Object} action The action Object
+ * @returns {Object} The new state.
+ */
+export default function search(state: UserSearchStateType = defState, action: Redux.Action<>) {
```

Comment:
Please add the Action type here, or remove the `<>`. I would prefer to add the action type, since it is only one action right now.